nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -1036,36 +1082,32 @@ impl<'a> BlueprintZonesBuilder<'a> {
 
     /// Produces an owned map of zones for the requested sleds
     pub fn into_zones_map(
-        mut self,
-        sled_ids: impl Iterator<Item = SledUuid>,
+        self,
+        commissioned_sled_ids: impl Iterator<Item = SledUuid>,
```

Comment:
Nitpick: There really isn't any part of this argument that validates that the state of these sled IDs as "commissioned", right?

Should this just be called "added_sleds"?

Reply:
Yeah, that does seem better. Renamed and expanded the doc comment in 64e866024